time_capture/time_capture/report/working_time/working_time.js

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) 2016, ALYF GmbH and contributors
+// For license information, please see license.txt
+/* eslint-disable */
+
+frappe.query_reports["Working Time"] = {
+	filters: [
+		{
+			fieldname: "year",
+			fieldtype: "Int",
+			label: "Year",
+			mandatory: 1,
+			wildcard_filter: 0,
+			default: moment().subtract(1, "month").year(),
+		},
+		{
+			fieldname: "month",
+			fieldtype: "Select",
+			label: "Month",
+			mandatory: 1,
+			options: [
+				{ value: 1, label: "January" },
```

Comment:
```suggestion
				{ value: 1, label: __("January") },
```

Should we translate the labels in this report?